modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -407,7 +407,11 @@ public ObjectEntry addObjectEntry(
 		_addFriendlyURLEntry(
 			objectDefinition, objectEntry, serviceContext, values);
 
-		_startWorkflowInstance(userId, objectEntry, serviceContext, false);
+		if (serviceContext.getWorkflowAction() ==
```

Comment:
Will this condition change the current behavior? Or is it just a way to avoid executing unnecessary code, since when it reaches [this point](https://github.com/liferay/liferay-portal/blob/01d3b69dea33528ace03bf6b6a7640beaa1c661b/portal-kernel/src/com/liferay/portal/kernel/workflow/WorkflowHandlerRegistryUtil.java#L79-L83) it stops?

Same question for the same change in the updateObjectEntry method.

Reply:
It's to avoid execute unnecessary code, but I will remove this change to do not pollute the code.

Thanks!!